style.css

```diff
@@ -0,0 +1,78 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 21px;
+}
+
+hr {
+  width: 95%;
+  border: 1px solid #212121;
+}
+
+/* header  */
+header {
+  background: #3c3a39;
+  width: 100%;
+  height: 11.2vh;
+  display: flex;
+}
+
+header h1 {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  display: flex;
+  gap: 200px;
+  align-items: center;
+  margin-left: 15px;
+}
+
+header img {
+  padding-left: 57vw;
+}
+
+/* intro */
+#intro {
+  background: #1c1a19;
+  width: 100%;
+  background-image: url("./img/intro-mobile.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  height: 167vh;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Lastly, the intro section seems to have the background image farther to the bottom of the page unlike what was in the Figma design I think this might be because of the height you gave to the section so I will advise that you remove the height completely or reduce the height so your design looks similar to the Figma design. Kindly ensure that your design looks similar to the Figma design as much as possible. :100:
- Your Design
![Screenshot from 2023-03-21 16-48-03](https://user-images.githubusercontent.com/102699958/226683751-d752de18-d184-4a68-bf9e-50690cc9a68d.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/226683906-a3b7cfdb-74ca-49a2-8239-91f2b37c6beb.png)